### DISCUSSION TITLE
localisation; configuration of display size for raspberry pi 7" (official display), multiroom sync [feature request]

### DISCUSSION THREAD

**Discussion by ffatgit** [jcorporation/myMPD#911] [Ideas]
Hi Mr. Mang, 
question/feature request no 1:
is there the possibility to include a localisation option into "myMPD".
If there are not plan's (or no time to do an integration in the next time), perhaps could you point me to the directory / files, what have to change. A example would be great!

question/feature request no 2:
I would like to use "myMPD" on a raspberry pi (based on dietpi) with a official 7" display. "myMPD" works right know, but for example in the "Playback" view the cover is a little bit to "big". The status bar with command button, the actual position in the track and the time stamp of the track, are "out" of the screen. I know I can scroll down, but it would be great, if either the whole UI could be "optimized" the the display size OR (for me personal) it would be a workaround to limit the size of the album art/cover.

question/feature request no 3:
Is there any plans to integrate "multiroom sync" (e.g. with the Snapcast server/client solution) into the UI. (https://github.com/badaix/snapcast)

Thank you for your great work so far and in advance for a feedback.
Frank

**Comment by jcorporation** [comment]
Hi Frank,

many thanks for your feedback and feature requests.

1) Localization is on the roadmap
2) What resolution have this display? Smaller than 300px width?
3) Integration of Snapcast is out of scope for myMPD. myMPD has the focus to support all features of mpd in a userfriendly way.

**Comment by ffatgit** [comment]
Hello Mr. Mang;

1. Nice to here (read) that the localization is "on the way". Is there a raw date to name?
2. The official raspberry pi display has a resolution of width x height = 800 x 480 px. And I don't understand the question for the width, because the height is the "problem".
3. Okay I understand and I thought already that those kind of answer is coming. No problem.

Thank you for the quick response!

**Comment by jcorporation** [comment]
1. There is no planed release date for this feature.

2. ok I misunderstood your request. You can edit the mympd.css file or add a custom css file to restrict the coverimage size.

**Comment by jcorporation** [comment]
myMPD 4.7.2 let's you configure the coverimagesize through mympd.conf
